### PR TITLE
Minor. Add session information log when session is created

### PR DESCRIPTION
**Pull request by jerryshao**
CC @zjffdu .

### CODE REVIEW

**Comment by jerryshao** [comment]
ping @zjffdu .

File: server/src/main/scala/com/cloudera/livy/server/interactive/InteractiveSession.scala [hunk 1]
```diff
@@ -421,6 +421,9 @@ class InteractiveSession(
 
       override def onJobSucceeded(job: JobHandle[Void], result: Void): Unit = {
         transition(SessionState.Running())
+        info(s"Interactive session $id created [appid: ${appId.orNull}, owner: $owner, proxyUser:" +
```
Comment (jerryshao) [inline server/src/main/scala/com/cloudera/livy/server/interactive/InteractiveSession.scala]: I also added the log before launching yarn app.

File: server/src/main/scala/com/cloudera/livy/server/interactive/CreateInteractiveRequest.scala [hunk 2]
```diff
@@ -36,4 +36,23 @@ class CreateInteractiveRequest {
   var name: Option[String] = None
   var conf: Map[String, String] = Map()
   var heartbeatTimeoutInSecond: Int = 0
+
+  override def toString: String = {
+    "Interactive session request:[" +
```
Comment (jerryshao) [inline server/src/main/scala/com/cloudera/livy/server/interactive/CreateInteractiveRequest.scala]: Done.

File: server/src/main/scala/com/cloudera/livy/server/batch/CreateBatchRequest.scala [hunk 3]
```diff
@@ -37,4 +37,22 @@ class CreateBatchRequest {
   var name: Option[String] = None
   var conf: Map[String, String] = Map()
 
+  override def toString: String = {
+    "Batch session request:[" +
```
Comment (jerryshao) [inline server/src/main/scala/com/cloudera/livy/server/batch/CreateBatchRequest.scala]: Done.

File: server/src/main/scala/com/cloudera/livy/server/interactive/InteractiveSession.scala (hunk 1 above)
Comment (zjffdu) [inline server/src/main/scala/com/cloudera/livy/server/interactive/InteractiveSession.scala]: Don't you log it after ping job succeeded ?

Comment (jerryshao) [inline server/src/main/scala/com/cloudera/livy/server/interactive/InteractiveSession.scala]: Yes, correct.

I'm not quite sure about your purpose.

Comment (zjffdu) [inline server/src/main/scala/com/cloudera/livy/server/interactive/InteractiveSession.scala]: sorry, misread your last comment. You have already log it before launching yarn app.

**Comment by zjffdu** [comment]
@jerryshao Can you help check the failed test ?

**Comment by zjffdu** [comment]
LGTM, merge it.

**Pull request merged**